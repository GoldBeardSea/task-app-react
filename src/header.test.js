import React from 'react';
import renderer from 'react-test-renderer';
import Header from "./header";

test('Renders Header', () => {
  const component = renderer.create(
    <Header>
    </Header>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});