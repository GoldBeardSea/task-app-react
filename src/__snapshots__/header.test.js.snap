// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Header 1`] = `
<header>
  <h1>
    Tasker
  </h1>
  <nav>
    Techformist
  </nav>
</header>
`;
